src/core/scene/a-scene.js

```diff
@@ -449,6 +450,46 @@ module.exports.AScene = registerElement('a-scene', {
       writable: window.debug
     },
 
+    setupCanvas: {
```

Comment:
We could move this out of the prototype to pure function. `setupCanvas (sceneEl)`.